src/containers/AppPage/Settings.js

```diff
@@ -0,0 +1,139 @@
+import { Component } from "react";
+import React from "react";
+import AppPage from "./index";
+
+// Color Palette
+// Dark Pink: #FF3576
+// Medium pink: #FF8FB3
+// Lighter pink: #F9DAE4
+// Sidebar pink: #FDF6F8
+// Grey: #EEEEEE
+
+// Circle: #FFAEC8
+// Lighter circle: #F9DAE4
+
+// Dark line: #AB4A78
+
+class Settings extends Component {
+    constructor() {
+        super()
+        this.state = {
+            currentBudget: "",
+            budgets: [
+              { month: 8, budget: 5000.00, spendings: 5510.00},
+              { month: 9, budget: 4000.00, spendings: 4567.89},
+              { month: 10, budget: 2500.00, spendings: 2500.00},
+              { month: 11, budget: 1500.00, spendings: 1000.00}
+           ]
+        }
+        this.handleChange = this.handleChange.bind(this)
+      }
+
+   handleChange(event) {
+       const {name, value} = event.target
+       this.setState({[name]: value})
+    }
+
+    handleSubmit(event) {
+       // if (event.target.password.value !== event.target.retype.value){
+            event.preventDefault()
+            alert("Your budget has been set.")
+            this.currentBudget.value ="";
+
+      // }
+       //else if (if form fields are empty){
+           //event.preventDefault()
+           //put alert
+       //}
+    //insert code to send information to database      
+    }
+
+    getDateMonth(monthNumber){
+      var month = new Array;
+      month[0] = "January";
+      month[1] = "February";
+      month[2] = "March";
+      month[3] = "April";
+      month[4] = "May";
+      month[5] = "June";
+      month[6] = "July";
+      month[7] = "August";
+      month[8] = "September";
+      month[9] = "October";
+      month[10] = "November";
+      month[11] = "December";
+      return month[monthNumber];
```

Comment:
This may be done with a simpler syntax such as, var month = ["January", "February",....] then accessed like month[x] i think?

Reply:
thank you yes! much simpler, I have changed to this :)